app/config/locale/translations/en.json

```diff
@@ -31,6 +31,20 @@
     "emails.mfaChallenge.clientInfo": "This verification code was requested using {{b}}{{agentClient}}{{/b}} on {{b}}{{agentDevice}}{{/b}} {{b}}{{agentOs}}{{/b}}. If you didn't request the verification code, you can safely ignore this email.",
     "emails.mfaChallenge.thanks": "Thanks,",
     "emails.mfaChallenge.signature": "{{project}} team",
+    "emails.mfaCreated.subject": "Two-factor authentication is enabled in your {{project}} account",
+    "emails.mfaCreated.hello": "Hello {{user}}",
+    "emails.mfaCreated.body": "Two-factor authentication has been successfully enabled for your {{project}} account.<br/><br/>Please make sure to store your security codes safely, so you can use them if you lose or can’t access your mobile device. Each recovery code can only be used once, but you can re-generate a new set of 6 codes at any time.",
```

Comment:
I'm not sure about adding the recovery code part because those are separate API calls that happen after. Also, if you delete and add, you may not go through the flow of generating the codes again.

Reply:
Not sure what to do here, this was the copy that was provided for cloud. Should we possibly split these into console/user facing?

Reply:
Probably, yes.